railties/lib/rails/commands/help/USAGE

```diff
@@ -9,6 +9,7 @@ The most common rails commands are:
 <% unless engine? -%>
  new          Create a new Rails application. "rails new my_app" creates a
               new application called MyApp in "./my_app"
+ plugin       Create a new Rails engine
```

Comment:
Technically, `rails plugin new my_plugin` creates a simple railtie, while `rails plugin new my_plugin --mountable` or `--full` creates an engine.  So, perhaps...

```suggestion
 plugin       Create a new Rails railtie or engine
```

Reply:
`rails plugin new -h | grep railtie` doesn't return any results.
The guides also don't seem to mention `Railtie` https://guides.rubyonrails.org/plugins.html
What do you think about...
```suggestion
 plugin       Create a new Rails plugin or engine
```

Reply:
I feel like "plugin" means "railtie or engine".  So if we prefer to not mention "railtie" explicitly, I think "Create a new Rails plugin" would suffice.  Although I should point out that `rails plugin new -h | grep engine` returns only the engine-specific options, and https://guides.rubyonrails.org/plugins.html doesn't mention "engine" at all (though it does hint at railtie, e.g. `require "yaffle/railtie"`).

Something else I just thought of -- should this be `plugin new`?

```suggestion
 plugin new   Create a new Rails engine
```

`rails plugin my_plugin` seems to be equivalent to `rails plugin new -h`.  (I wonder if we can / should change that? :thinking:)

Reply:
Hmm, good points.

The engine guides does mention `railtie` be describing `bin/rails railties:install:migrations` 
https://guides.rubyonrails.org/engines.html. Plugin does seem the superset of railtie and engine.

The current description of the plugin command also describes it as `rails plugin new`.
```
Description:
    The 'rails plugin new' command creates a skeleton for developing any
    kind of Rails extension with ability to run tests using dummy Rails
    application.
```

Reply:
So my new preference would be:
```suggestion
 plugin new       Create a new Rails railtie or engine
```